cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -370,7 +370,7 @@ export class Application {
             .within(() => {
                 click(selectBox);
                 cy.wait(SEC);
-                click('button[aria-label="Actions"]');
+                click('button[aria-label="Kebab toggle"]');
```

Comment:
Igor please note this in the spreadsheet that we have for UI selectors . We can ask dev to add and ID here .

Reply:
I will add it